modules/hitlnext/src/views/lang.ts

```diff
@@ -0,0 +1,33 @@
+import lang from 'common/lang'
+
+import en from '../translations/en.json'
+import fr from '../translations/fr.json'
+
+const defaultLocale = 'en'
+const translations = { fr, en }
+let locale = ''
+let isDev: boolean
+
+const init = () => {
+  lang.init(translations, 'module.hitlnext.')
+  locale = getUserLocale()
+  isDev = localStorage.getItem('langdebug') === 'true'
+}
+
+const getUserLocale = () => {
+  const code = str => str.split('-')[0]
+  const browserLocale = code(navigator.language || navigator['userLanguage'] || '')
+  const storageLocale = code(localStorage.getItem('uiLanguage') || '')
+
+  return translations[storageLocale] ? storageLocale : translations[browserLocale] ? browserLocale : defaultLocale
+}
+
+const tr = (key: string, values?: { [variable: string]: any }) => {
+  if (isDev) {
+    return key
+  }
+
+  return lang.tr(key, values, locale || defaultLocale)
+}
+
+export default { tr, init }
```

Comment:
Translations are handled by the studio, this should not be necessary. The studio fetches all translations from the backend on load.

![image](https://user-images.githubusercontent.com/42552874/100128984-f48adc80-2e4e-11eb-8598-eb66f51610ce.png)

Reply:
Nop they're not ... that's for custom components injected in channel-web. Translations are not initialized.